compiler/luci/pass/include/luci/Pass/QuantizeWithMinMaxPass.h

```diff
@@ -31,6 +31,17 @@ namespace luci
  */
 class QuantizeWithMinMaxPass : public logo::Pass
 {
+public:
+  struct Context
+  {
+    loco::DataType input_model_dtype;
```

Comment:
(optional) OK with no default?

Reply:
AFAIK, we do not give initial value to loco::DataType. Unknown is a default value of loco::DataType.